qutebrowser/config/configtypes.py

```diff
@@ -798,6 +799,18 @@ class File(BaseType):
 
     typestr = 'file'
 
+    def transform(self, value):
+        if not value:
+            return None
+        value = os.path.expanduser(value)
+        value = os.path.expandvars(value)
+        if not os.path.isabs(value):
+            if standarddir.config() is not None:
+                abspath = os.path.join(standarddir.config(), value)
+                if os.path.isfile(abspath):
```

Comment:
Oh yes. I think I've been a bit too caught up in trying to fix what you point out that I haven't looked at the bigger picture. I'll take a bit more time to review it this time, sorry. Also, I've got exams coming up, so it may take a bit longer.